cmd/notation/verify.go

```diff
@@ -155,13 +157,29 @@ func runVerify(command *cobra.Command, opts *verifyOpts) error {
 
 func checkVerificationFailure(outcomes []*notation.VerificationOutcome, printOut string, err error) error {
 	// write out on failure
-	if err != nil || len(outcomes) == 0 {
```

Comment:
Should we keep the check of `len(outcomes) == 0`?

Reply:
Yeah, I added it back.
This is needed when `err` is nil. Because when `err` is nil, it means a successful verification with length of `outcomes` being exactly 1.